core/src/main/java/org/bitcoinj/core/LockTime.java

```diff
@@ -131,46 +119,12 @@ public long rawValue() {
     }
 
     /**
-     * The lock time is considered to be set only if its raw value is greater than zero.
+     * This is equivalent to {@code lockTime instanceof HeightLock || lockTime instanceof TimeLock}.
```

Comment:
Really? I thought "unset" is same as a HeightLock of 0?

Reply:
Oops. This was an error. Fixed.